src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,7 @@
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        User user = UserService.findByEmail(email);
```

Comment:
does your code compile?
we should create an object of UserService, and call `findByEmail()` on this object